IntroducingMethods/src/cst1201/IntroducingMethods.java

```diff
@@ -3,33 +3,75 @@
 import java.util.Scanner;
 
 /**
- * TODO: Description of your class.
+ * Description of your class.
  *
- * @author TODO: Your name here.
+ * @author Francisco Latacela and Goutam Kundu .
  */
 public class IntroducingMethods {
 
     public static void main(String[] args) {
 
-        // TODO: Fill in the body with your code
+        //Fill in the body with your code
+        System.out.println("Please Enter a String: ");
+        Scanner sc = new Scanner(System.in);
+        getInputString(sc);
     }
 
-    // Given a Scanner, prompt the user for a String.  If the user enters an empty
-    // String, report an error message and ask for a non-empty String.  Return the
-    // String to the calling program.
+    /**
+     * Given a Scanner, prompts the user for a String. If the user enters an
+     * empty String, reports an error message and asks for a non-empty String.
+     * Returns the String to the calling program.
+     *
+     * @param inScanner The Scanner to use to prompt the user.
+     * @return The resulting String from the user.
+     */
     private static String getInputString(Scanner inScanner) {
-        // TODO: Fill in the body
-
-        // NOTE: Do not declare a Scanner in the body of this method.
+        //Fill in the body
+        String input = inScanner.nextLine();
+        while (input.length() == 0) {
+            System.out.println("You have entered an empty String, Please enter a non-empty String");
+            System.out.println("Please Enter a String: ");
+            input = inScanner.nextLine();
+        }
+        getWordCount(input);
```

Comment:
Nope. `getInputString()` only gets an input string and that's it. Please see the description.